core/src/css/test/theme-ionic/typography.e2e.ts

```diff
@@ -0,0 +1,185 @@
+            font-weight: 100;
+            margin-bottom: 8px;
+            padding: 8px;
+          }
+        </style>
+
+        <ion-content class="ion-padding" id="content">
+          <ion-text class="ionic-preview">Utility Classes - Headings</ion-text>
+          <p><ion-text class="ionic-heading1">Utility class 'ionic-heading1'</ion-text></p>
+          <p><ion-text class="ionic-heading2">Utility class 'ionic-heading2'</ion-text></p>
+          <p><ion-text class="ionic-heading3">Utility class 'ionic-heading3'</ion-text></p>
+          <p><ion-text class="ionic-heading4">Utility class 'ionic-heading4'</ion-text></p>
+          <p><ion-text class="ionic-heading5">Utility class 'ionic-heading5'</ion-text></p>
+          <p><ion-text class="ionic-heading6">Utility class 'ionic-heading6'</ion-text></p>
+        </ion-content>
+      `,
+        config
+      );
+
+      const globalStyles = page.locator('ion-content');
+      await expect(globalStyles.nth(0)).toHaveScreenshot(screenshot(`ionic-utility-classes-headings`));
+    });
+
+    test('should apply the utility classes for font weights', async ({ page }) => {
+      await page.setContent(
+        `
+        <link href="../../../../../css/ionic.globals.ionic.css" rel="stylesheet" />
+        <style>
+          .ionic-preview {
+            background: #d7d8da;
+            border-radius: 16px;
+            box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+            font-size: 16px;
+            font-weight: 100;
+            margin-bottom: 8px;
+            padding: 8px;
+          }
+        </style>
+
+        <ion-content class="ion-padding" id="content">
+          <ion-text class="ionic-preview">Font Weights</ion-text>
+          <p><ion-text class="ionic-font-light">Utility class 'ionic-font-light'</ion-text></p>
+          <p><ion-text class="ionic-font-regular">Utility class 'ionic-font-regular'</ion-text></p>
+          <p><ion-text class="ionic-font-medium">Utility class 'ionic-font-medium'</ion-text></p>
+          <p><ion-text class="ionic-font-semibold">Utility class 'ionic-font-semibold'</ion-text></p>
+          <p><ion-text class="ionic-font-bold">Utility class 'ionic-font-bold'</ion-text></p>
+        </ion-content>
+      `,
+        config
+      );
+
+      const globalStyles = page.locator('ion-content');
+      await expect(globalStyles.nth(0)).toHaveScreenshot(screenshot(`ionic-utility-classes-font-weights`));
+    });
+
+    test('should apply the utility classes for font display', async ({ page }) => {
+      await page.setContent(
+        `
+        <link href="../../../../../css/ionic.globals.ionic.css" rel="stylesheet" />
+        <style>
+          .ionic-preview {
+            background: #d7d8da;
+            border-radius: 16px;
+            box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+            font-size: 16px;
+            font-weight: 100;
+            margin-bottom: 8px;
+            padding: 8px;
+          }
+        </style>
```

Comment:
Remove this snippet. We try to not include any additional styling in order to keep the snapshots as clean as possible. If we added additional styling, it could lead to misleading snapshots that may not align with the original code.

```suggestion
```

Reply:
All the styles for preview were removed